src/data-fetchers/bam/bam-worker.ts

```diff
@@ -1,12 +1,13 @@
 // This worker is heavily based on https://github.com/higlass/higlass-pileup/blob/master/src/bam-fetcher-worker.js
```

Comment:
We could remove the line or change the comment?

```suggestion
// Adopted from https://github.com/higlass/higlass-pileup/blob/master/src/bam-fetcher-worker.js
```